[Oracle] README updates regarding mandatory user permission (#9104)
elastic/integrations: packages/oracle/_dev/build/docs/README.md

Tablespace metrics describes the tablespace usage metrics of all types of tablespaces in the oracle database.

> To collect the Tablespace metrics, Oracle integration relies on a specific set of views. It is important to verify that the user configured within the Oracle DSN configuration has `READ` access permissions to the views listed below.

Reviewer: ```suggestion
To collect the Tablespace metrics, Oracle integration relies on a specific set of views. Make sure that the user configured within the Oracle DSN configuration has `READ` access permissions to the following views:
```
For accessibility purposes, terms like "below" or "above" that indicate directions should be avoided.
Check also [How to create accessible web content](https://docs.google.com/document/d/1IGuMf16-cwxx8BX1Z9PWsFWsTPxicwTkX_r7UOhjd4E/edit#heading=h.iykpo698d18b).

 
- `SYS.DBA_DATA_FILES`
- `SYS.DBA_TEMP_FILES`
- `DBA_FREE_SPACE`

{{fields "tablespace"}}

{{event "tablespace"}}

### Sysmetrics 

> The system metrics value captured for the most current time interval for the long duration (60-seconds) are mentioned below. 

Reviewer: ```suggestion
The system metrics value captured for the most current time interval for the long duration (60-seconds) are listed in the following table. 
```

> To collect the Sysmetrics metrics, Oracle integration relies on a specific set of views. It is important to verify that the user configured within the Oracle DSN configuration has `READ` access permissions to the view mentioned below.

Reviewer: ```suggestion
To collect the Sysmetrics metrics, Oracle integration relies on a specific set of views. Make sure that the user configured within the Oracle DSN configuration has `READ` access permissions to the following view:
```

A Program Global Area (PGA) is a memory region that contains data and control information for a server process. It is nonshared memory created by Oracle Database when a server process is started. Access to the PGA is exclusive to the server process. Metrics concerning Program Global Area (PGA) memory are mentioned below.

> To collect the Memory metrics, Oracle integration relies on a specific set of views. It is important to verify that the user configured within the Oracle DSN configuration has `READ` access permissions to the views listed below.

Reviewer: ```suggestion
To collect the Memory metrics, Oracle integration relies on a specific set of views. Make sure that the user configured within the Oracle DSN configuration has `READ` access permissions to the following views:
```

The System Global Area (SGA) is a group of shared memory structures that contain data and control information for one Oracle Database instance. Metrics concerning System Global Area (SGA) memory are mentioned below.

> To collect the System Statistics metrics, Oracle integration relies on a specific set of views. It is important to verify that the user configured within the Oracle DSN configuration has `READ` access permissions to the view mentioned below.

Reviewer: ```suggestion
To collect the System Statistics metrics, Oracle integration relies on a specific set of views. Make sure that the user configured within the Oracle DSN configuration has `READ` access permissions to the following view:
```

Performance metrics give an overview of where time is spent in the system and enable comparisons of wait times across the system.

> To collect the Performance metrics, Oracle integration relies on a specific set of views. It is important to verify that the user configured within the Oracle DSN configuration has `READ` access permissions to the views listed below.

Reviewer: ```suggestion
To collect the Performance metrics, Oracle integration relies on a specific set of views. Make sure that the user configured within the Oracle DSN configuration has `READ` access permissions to the following views:
```
